src/main/java/org/folio/innreach/batch/contribution/service/InitialContributionMessageListener.java

```diff
@@ -0,0 +1,28 @@
+package org.folio.innreach.batch.contribution.service;
+
+import lombok.AllArgsConstructor;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.folio.innreach.batch.contribution.ContributionJobContext;
+import org.folio.innreach.domain.dto.folio.inventorystorage.InstanceIterationEvent;
+import org.springframework.kafka.listener.AcknowledgingMessageListener;
+import org.springframework.kafka.support.Acknowledgment;
+
+@AllArgsConstructor
+public class InitialContributionMessageListener implements AcknowledgingMessageListener<String, InstanceIterationEvent> {
```

Comment:
See my comment on previous versions. Should this be implementing this interface?